ffs/scheduler/scheduler.go

```diff
@@ -422,7 +422,9 @@ func (s *Scheduler) executeQueuedStorage(j ffs.StorageJob) {
 
 	// Execute
 	s.l.Log(ctx, "Executing job %s...", j.ID)
-	info, dealErrors, err := s.executeStorage(ctx, a, j)
+	dealUpdates := s.sjs.MonitorJob(j)
+	info, dealErrors, err := s.executeStorage(ctx, a, j, dealUpdates)
```

Comment:
get the channel from sjstore and pass it down into the job execution/cold storage execution layer.